src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -718,31 +718,28 @@ def handle_migration_parameters(command_group, server_name_arg_type, migration_i
                                help='Request properties. Use double or no quotes to pass in filepath as argument.')
                     c.argument('migration_name', arg_type=migration_id_arg_type, options_list=['--migration-name'],
                                help='Name of the migration.')
+                    c.argument('migration_mode', arg_type=migration_id_arg_type, options_list=['--migration-mode'], required=False,
+                               help='Either offline or online(with CDC) migration', choices=['offline', 'online'], default='offline')
                 elif scope == "show":
                     c.argument('migration_name', arg_type=migration_id_arg_type, options_list=['--migration-name'],
                                help='Name of the migration.')
-                    c.argument('level', options_list=['--level'], required=False,
-                               help='Specify the level of migration details requested. Valid values are Active and All. Active is the default.')
```

Comment:
Will these parameter changes take effect for both `postgres flexible-server migration` and `mysql flexible-server migration`?